### DISCUSSION TITLE
Could this Package Be Used for a Smart Lawnmower?

### DISCUSSION THREAD

**Discussion by Koichi-san-hu** [Fields2Cover/Fields2Cover#105] [General]
Hello everyone, do you think this package could be used to turn a lawnmower into a smart one? I already have nav2 working on my lawnmower and I'm going crazy trying to implement a full cover path. Could this be a good solution?"

**Comment by Gonzalo-Mier** [comment]
Yes, for sure. Just make sure to keep changing the path from time to time to avoid visible footprints on your land :)

**Reply by tlschumacher2000** [reply]
Could a randomized width factor be used to alter the width slightly?  If I have a 24 inch mower head, Swath 1 is normal, Swath 2 has an overlap on Run 1 of 2 inches.  On Run 2, it has an overlap of 3 inches.  If this is repeated on each swath, each Run would follow a slightly different path.  It was something I was looking into a few years ago before I found ROS.

**Reply by Gonzalo-Mier** [reply]
Not my field, but I have the feeling that mowing always in the same direction would end up with some visible marks on the grass.
